php/shared/usr/local/share/assets/assets_functions.sh

```diff
@@ -162,33 +162,35 @@ function assets_apply_database_postgres()
   wait_for_remote_ports "${ASSETS_DATABASE_WAIT_TIMEOUT}" "${DATABASE_HOST}:${DATABASE_PORT}"
 
   local DATABASES
-  mapfile -t DATABASES < <(PGPASSWORD="$PGPASSWORD" psql "${DATABASE_ARGS[@]}" -lqt | cut -d \| -f 1 | sed "s/ //g")
+  DATABASES="$(set -o pipefail && PGPASSWORD="$PGPASSWORD" psql "${DATABASE_ARGS[@]}" -lqt | cut -d \| -f 1 | sed "s/ //g")"
 
   set +e
-  ! printf "%s\\n" "${DATABASES[@]}" | grep --quiet --fixed-strings --line-regexp "${APPLY_DATABASE_NAME}"
+  ! echo "${DATABASES}" | grep --quiet --fixed-strings --line-regexp "${APPLY_DATABASE_NAME}"
   local DATABASE_EXISTS=$?
   set -e
 
-  local DATABASE_TABLES=()
+  local DATABASE_TABLE_COUNT=0
   if [ "${DATABASE_EXISTS}" -ne 0 ]; then
-    mapfile -t DATABASE_TABLES < <(PGPASSWORD="$PGPASSWORD" psql "${DATABASE_ARGS[@]}" -c '\dt' -qt "${APPLY_DATABASE_NAME}" | cut -d \| -f 1 | sed "s/ //g")
+    DATABASE_TABLES="$(set -o pipefail && PGPASSWORD="$PGPASSWORD" psql "${DATABASE_ARGS[@]}" -c '\dt' -qt "${APPLY_DATABASE_NAME}" | cut -d \| -f 1 | sed "s/ //g")"
```

Comment:
Does this need declaring as a local variable still?

Reply:
yes, fixed